ReportUtilityBills/measure.rb

```diff
@@ -47,16 +47,28 @@ def arguments(model = nil) # rubocop:disable Lint/UnusedMethodArgument
 
     arg = OpenStudio::Measure::OSArgument::makeBoolArgument('include_annual_bills', false)
     arg.setDisplayName('Generate Annual Utility Bills')
-    arg.setDescription('Generates annual utility bills.')
+    arg.setDescription('Generates output file containing annual utility bills.')
+    arg.setDefaultValue(true)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('report_annual_bills', false)
+    arg.setDisplayName('Report Annual Utility Bills')
+    arg.setDescription('Reports annual utility bills.')
     arg.setDefaultValue(true)
     args << arg
 
     arg = OpenStudio::Measure::OSArgument::makeBoolArgument('include_monthly_bills', false)
     arg.setDisplayName('Generate Monthly Utility Bills')
-    arg.setDescription('Generates monthly utility bills.')
+    arg.setDescription('Generates output file containing monthly utility bills.')
     arg.setDefaultValue(true)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('report_monthly_bills', false)
+    arg.setDisplayName('Report Monthly Utility Bills')
+    arg.setDescription('Reports monthly utility bills.')
```

Comment:
This needs more clarification, I don't think anyone will understand what "report" means. How about something like:
```
    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('register_monthly_bills', false)
    arg.setDisplayName('Register Monthly Utility Bills')
    arg.setDescription('Registers monthly utility bills with the OpenStudio runner for downstream processing.')
```